src/sssom/parsers.py

```diff
@@ -182,6 +182,95 @@ def _get_seperator_symbol_from_file_path(file):
     return None
 
 
+def _is_check_valid_extension_slot(slot_name, meta):
+    extension_definitions = meta.get("extension_definitions", [])
+    return any(entry.get("slot_name") == slot_name for entry in extension_definitions)
```

Comment:
I would personally like a check that the extension definition does have a `property` (which it SHOULD have), and a warning if it does not.

Reply:
Done! https://github.com/mapping-commons/sssom-py/commit/57d24f9bbdaf7b640e0d9d7dc6e4ff04cdaf9ec2